ccpp/data/GFS_typedefs.F90

```diff
@@ -2867,6 +2867,8 @@ subroutine coupling_create (Coupling, IM, Model)
 !   endif
 
     if (Model%cplflx .or. Model%cpllnd) then
+      allocate (Coupling%dlwsfci_cpl (IM))
```

Comment:
Since the conditions for which `dswsfci_cpl` and `dlwsfci_cpl` are changing, a commensurate change should be made in GFS_typedefs.meta. Look for the entries for these variables in that file, and for their active attribute, instead of `active = (flag_for_surface_flux_coupling)`, we need `active = (flag_for_surface_flux_coupling .or. flag_for_land_coupling)`.

Reply:
@grantfirl Thanks. I've made the change in the meta data now.